cuda/python/python3.10/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ghcr.io/neuralmagic/base-cuda11.1:latest
+
+RUN set -Eeuxo \
```

Comment:
Let's combine the two RUN statements, to allow use fresh apt-get update metadata